tools/flake8.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+#
+# run flake8 with project settings
+
+set -eu
```

Comment:
What does this file add?  I just do this:
```sh
poetry install
poetry shell
pflake8 .
```
I think we need complicated scripts for people who don't know how to use poetry.

Reply:
Removed scripts.